releasenotes/notes/websocket-span-pointers-25e07939aa75527a.yaml

```
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+     feat(asgi): Enable context propagation between websocket message spans by implementing span pointers between incoming and outgoing messages.
```

is "span pointers" a feature customers should be aware of?

you can probably just shorten to:


```suggestion
     feat(asgi): Enable context propagation between websocket message spans.
```

or maybe


```suggestion
     feat(asgi): Enable context propagation between websocket incoming and outgoing message spans.
```

if the incoming/outgoing context is still useful.


since I assume "span pointers" is just an implementation detail and not something most customers need to care about?